network/common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -105,4 +105,19 @@ public boolean lazyFileDescriptor() {
   public int portMaxRetries() {
     return conf.getInt("spark.port.maxRetries", 16);
   }
+
+  /**
+   * Maximum number of bytes to be encrypted at a time when SASL encryption is enabled.
+   */
+  public int maxSaslEncryptedBlockSize() {
+    return conf.getInt("spark.network.sasl.max_encrypted_block_size_kb", 64) * 1024;
```

Comment:
nit: spark config style is camelCase for things which are not separated by periods